modules/ROOT/pages/6.3-release-notes.adoc

```diff
@@ -0,0 +1,694 @@
+
+[source,text]
+----
+"They went that way," he said, pointing.
+----
+
+Selecting _English (US)_ from the *Advanced Typography* toolbar icon’s pop-up menu transforms the text, as expected, into
+
+[source,text]
+----
+“They went that way,” he said, pointing.
+----
+
+Following this action by, then, selecting _Spanish_ from the *Advanced Typography* toolbar icon’s pop-up menu, however, can result in the following incomplete typesetting change:
+
+[source,text]
+----
+«They went that way, ” he said, pointing.
+----
+
+As of this release the only workaround is to not change *Advanced Typography* target languages after running any given target language’s rules over a {productname} document.
+
+[[accompanying-premium-plugin-changes]]
+== Accompanying Premium Plugin changes
+
+The following premium plugin updates were released alongside {productname} 6.3.
+
+=== Accessibility Checker 3.1.0
+
+The {productname} 6.3 release includes an accompanying release of the **Accessibility Checker** premium plugin.
+
+**Accessibility Checker** 3.1.0 includes the following fixes, additions, and improvements.
+
+==== Merge Tags prefix and suffix elements were not ignored
+
+Previously, **Accessibility Checker** incorrectly flagged xref:mergetags.adoc[Merge Tags] prefixes and suffixes as presenting a contrast issue.
+
+With this update, the **Accessibility Checker** plugin ignores Merge Tag prefixes and suffixes, as expected.
+
+==== Added a new rule to show a warning when the alternative text of an image exceeds 100 characters
+
+**Accessibility Checker** 3.1.0 now includes a rule that, when applied, shows a warning dialog when an image’s alternative (`+alt+`) text attribute exceeds 100 characters.
+
+When the alternative text exceeds 100 characters, an *Image alternative text should be less than 100 characters* warning presents.
+
+This dialog also presents the alternative text in an editable field, for immediate repair.
+
+==== The discoverability of the question mark button in warning dialogs was improved
+
+Many **Accessibility Checker** dialogs include a question mark icon. This icon is a link to further information about the discovered issue.
+
+This update includes changes to the presentation of this icon to make discoverability as a clickable button more apparent.
+
+==== The legibility of warning dialogs was improved
+
+In concert with the changes noted above to the question mark icon in **Accessibility Checker** dialogs, legibility improvements were made to the presentation and typography of these dialogs.
+
+The icon’s text was centered and the *Issue*, *Warning*, or *Error* sub-heading’s font size was reduced.
+
+Additionally, borders were removed from the exterior of the description field and contrast colors were adjusted to match the background color of the icons added for this release.
+
+==== Replaced the single warning icon presented in dialogs with icons specific to each alert level;
```

Comment:
The heading line has a semi-colon at the end - this appears inconsistent with the other similar lines.

Reply:
```suggestion
==== Replaced the single warning icon presented in dialogs with icons specific to each alert level
```